presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -160,7 +160,7 @@
     private final Set<InPredicate> subqueryInPredicates = newIdentityHashSet();
     private final Set<Expression> columnReferences = newIdentityHashSet();
     private final IdentityHashMap<Expression, Type> expressionTypes = new IdentityHashMap<>();
-    private final Set<QuantifiedComparisonExpression> subqueryQuantifiedComparisons = newIdentityHashSet();
+    private final Set<QuantifiedComparisonExpression> quantifiedComparisons = newIdentityHashSet();
```

Comment:
Use the name "quantifiedComparisons" in the commit that introduce the field. No point in adding it with one name just to rename it in the commit that follows.